examples/graphql-ws/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { createServer } from '@graphql-yoga/node'
+import { WebSocketServer } from 'ws'
+import { useServer } from 'graphql-ws/lib/use/ws'
+
+async function main() {
+  const yogaApp = createServer({
+    graphiql: {
+      subscriptionsProtocol: 'WS',
```

Comment:
Is `subscriptionsProtocol` the best pick here?
Also, the value should be `graphql-transport-ws` IMHO, as it is the same as [specified in the spec](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#communication).

Last but not least, what if I want to also execute mutation and query operations via the `graphql-transport-ws` protocol? This is currently not supported?